packages/xchain-cosmos/src/client.ts

```diff
@@ -29,14 +29,20 @@ export interface CosmosClient {
   getSDKClient(): CosmosSDKClient
 }
 
-const MAINNET_SDK = new CosmosSDKClient({
-  server: 'https://api.cosmos.network',
-  chainId: 'cosmoshub-4',
-})
-const TESTNET_SDK = new CosmosSDKClient({
-  server: 'https://rest.sentry-02.theta-testnet.polypore.xyz',
-  chainId: 'theta-testnet-001',
-})
+export interface CosmosClientParams extends XChainClientParams {
+  serverUrl?: string
```

Comment:
Add a parameter to set `serverUrl` for all possible `Network`s (including `testnet` etc.). Similar to `xchain-thorchain` or other packages.

Reply:
Fixed in #599